pkg/azurefile/azurefile.go

```diff
@@ -38,6 +38,7 @@ const (
 	DriverName       = "file.csi.azure.com"
 	seperator        = "#"
 	volumeIDTemplate = "%s#%s#%s"
+	fileUrlTemplate  = "https://%s.file.core.windows.net"
```

Comment:
need to use `cloud.Environment` to support other cloud, e.g. Azure China

Reply:
For mooncake, it's `https://%s.file.core.chinacloudapi.cn`, so it should be "https://%s.file." + `cloud.Environment.StorageEndpointSuffix`

refer to
https://github.com/Azure/go-autorest/blob/403aff94c3536f190b71fde971cb66963202f6d3/autorest/azure/environments.go#L157

Reply:
done.